steward/src/cork.rs

```diff
@@ -31,6 +35,10 @@ pub mod proposals;
 const MESSAGE_TIMEOUT: Duration = Duration::from_secs(10);
 const CHAIN_PREFIX: &str = "somm";
 
+lazy_static! {
+    static ref STEWARD_VERSION: &'static str = env!("CARGO_PKG_VERSION");
+};
```

Comment:
I believe all the tests are failing because this line doesn't compile. We should fix and verify that the status endpoint works in practice.

Reply:
Sorry, was moving too fast. Should have caught this.